testing/matrix-sdk-test/src/test_json/keys_query_sets.rs

```diff
@@ -486,7 +486,7 @@ impl IdentityChangeDataSet {
         })
     }
 
-    fn msk_a() -> Value {
+    pub fn msk_a() -> Value {
```

Comment:
Not your fault, but we avoid the MSK abbreviation in the SDK. No need to rename this but I wanted to remind people.

Reply:
:+1:  Will update in a later PR.